framework/wazuh/core/stats.py

```diff
@@ -117,6 +117,46 @@ def totals_(date=utils.get_utc_now()):
     return affected
 
 
+def get_daemons_stats_socket(socket: str) -> dict:
+    """Send message to Wazuh socket to get statistical information.
+
+    Parameters
+    ----------
+    socket : str
+        Full path of the socket to communicate with.
+
+    Raises
+    ------
+    WazuhInternalError (1121)
+        If there was an error when trying to connect to the socket.
+
+    Returns
+    -------
+    dict
+        Dictionary with daemon's statistical information.
+    """
+    # Create message
+    full_message = wazuh_socket.create_wazuh_socket_message(origin={'module': common.origin_module.get()},
+                                                            command='getstats')
+
+    # Connect to socket
+    try:
+        s = wazuh_socket.WazuhSocketJSON(socket)
+    except Exception:
+        raise WazuhInternalError(1121, extra_message=socket)
+
+    # Send message and receive socket response
+    s.send(full_message)
+    response = s.receive()
+    s.close()
```

Comment:
Always close the connection on `finally` block or using context manager if possible.

Reply:
done